PIL/ImageShow.py

```diff
@@ -87,7 +87,7 @@ def get_command(self, file, **options):
 
     def save_image(self, image):
         """Save to temporary file, and return filename"""
-        return image._dump(format=self.get_format(image))
+        return image._dump(format=self.get_format(image), compress_level=1)
 
```

Comment:
Look - I don't want to re-engineer a "perfect" solution and rewrite the whole thing. There was a problem: down-convert strip transparency and mid-displays. The compress_level thing is just a nice addition: if it would works, good, but display slowly and correctly is still better than fast and wrongly.

What I mean is that if the compress_level change takes too much effect/discussion, I 'd say just do without it. So this series is logically three changes, use transparency-capable intermediate format, do not down-convert, and make transparency-format works well. If the 3Rd cannot be done, or takes too much discussion, I 'd say just live without it.